front/app/components/PostShowComponents/Comments/index.tsx

```diff
@@ -174,7 +157,7 @@ const CommentsSection = memo<Props>(
       return (
         <Container className={className || ''}>
           <Header>
-            <Title id="comments-main-title">
+            <Title variant="h2" id="comments-main-title">
```

Comment:
This is great! Let's just bump the component library and add the color `colorText`.